docs/elements.md

```diff
@@ -931,6 +931,7 @@ Provides an in-browser rich text editor, aimed mostly at manual grading essay-ty
 | `markdown-shortcuts` | boolean                             | true                 | Whether or not the editor accepts shortcuts based on markdown format (e.g., typing `_word_` causes the word to become italic).                                                                                                                                                                                                                                                                                                                            |
 | `counter`            | `"word"`, `"character"` or `"none"` | `"none"`             | Whether a word or character count should be displayed at the bottom of the editor.                                                                                                                                                                                                                                                                                                                                                                        |
 | `allow-blank`        | boolean                             | false                | Whether or not an empty input box is allowed. By default, empty submissions will not be graded (invalid format).                                                                                                                                                                                                                                                                                                                                          |
+| `clipboard-enabled`  | boolean                             | true                 | Whether or not the element supports cutting, copying and pasting the contents of the editor.                                                                                                                                                                                                                                                                                                                                                              |
```

Comment:
I can leave a full review on Monday, but: Matt and I had discussed calling out that this setting is detrimental to the student experience and should be avoided unless it's absolutely necessary. IMO we should do that.

Reply:
That's a good point. I'll also point out that it's impossible to fully restrict students from using the feature, as described here for `pl-code`'s `prevent-select`:
https://github.com/PrairieLearn/PrairieLearn/blob/445df9d41de610aada770f7c27b10698c9e3b5b5/docs/elements.md#L1197

Should I add your warning to pl-code as well?

Reply:
Done both in d4849ad47639b53a933fee911bd7c4842678a2c8.